package-lock.json

```diff
@@ -9519,6 +9524,14 @@
       "integrity": "sha512-wK0Ri4fOGjv/XPy8SBHZChl8CM7uMc5VML7SqiQ0zG7+J5Vr+RMQDoHa2CNT6KHUnTGIXH34UDMkPzAUyapBZg==",
       "dev": true
     },
+    "test": {
```

Comment:
after removing `test` from `package.json` this lock file has to be regenerated to remove the change completely, with `npm install`